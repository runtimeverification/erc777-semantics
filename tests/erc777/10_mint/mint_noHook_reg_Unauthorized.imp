test.preamble();

// operatorMint() through default operator (unless otherwise specified), operatorMint() from caller.
// regular recipient address with no hook, minting through unauthorized operator

test.makeCaller(5);
test.operatorMint(6, 1000, "", ""); //through default operator
