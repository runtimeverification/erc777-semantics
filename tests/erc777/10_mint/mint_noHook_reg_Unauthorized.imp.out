Changing the caller to 5
5 is minting 1000 for 6 (data:""),  (operatorData:"")
REVERT
Balance of 6 is 60
Total supply is 550
